go.mod

```diff
@@ -57,6 +57,8 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+replace github.com/redhat-appstudio/release-service => github.com/davidmogar/release-service v0.0.0-20230926115200-6b1480ca0688
```

Comment:
What's the plan here? To move back to the main repo before merging this, once the release-service changes are merged?

Reply:
Yes.